libbeat/docs/dashboardsconfig.asciidoc

```diff
@@ -41,7 +41,9 @@ You can specify the following options in the `setup.dashboards` section of the
 
 If this option is set to true, {beatname_uc} loads the sample Kibana dashboards
 automatically on startup. If no other options are set, the dashboard are loaded
-from the local `kibana` directory in the home path of the installation.
+from the local `kibana` directory in the home path of the installation. Note that
```

Comment:
This seems fairly important, so you might want to call this out as a separate note. I wonder if it might make sense to have a `setup.dashboards.overwrite` option like we have with the index template?

I'd suggest the following changes (includes a few edits to remove ambiguity in the existing text): 

```
If this option is set to true, {beatname_uc} loads the sample Kibana dashboards
from the local `kibana` directory in the home path of the {beatname_uc} installation.

NOTE: When dashboard loading is enabled, {beatname_uc} overwrites any existing
dashboards that match the names of the dashboards you are loading. This happens
every time {beatname_uc} starts. 

```

Reply:
@dedemorton https://github.com/elastic/beats/blob/9e2838d9ecaf4d9c3ea05ba7f97182cdd2d93574/libbeat/dashboards/kibana_loader.go#L89 suggests we indeed force overwriting the dashboards (not a developer on Beats, could be wrong about this).

I opened a Discuss on this: https://discuss.elastic.co/t/should-beats-modules-overwrite-kibana-dashboards/127984

I like the new wording. I patched the PR.